src/App.js

```diff
@@ -1,5 +1,65 @@
+import { Console } from '@woowacourse/mission-utils';
+import Racer from './Racer.js';
+import Racing from './Racing.js';
+import {
+  FIRST_USER_INPUT_PLACEHOLDER,
+  SECOND_USER_INPUT_PLACEHOLDER,
+  NAME_SEPARATOR,
+  OUTPUT_PREFIX,
+} from './constants/index.js';
+import Winners from './Winners.js';
+import Validator from './Validator.js';
+
 class App {
-  async run() {}
+  #totalRound = 0;
+
+  #racers = [];
+
+  async run() {
+    await this.#setRacers();
+    await this.#setTotalRound();
+
+    this.#startRacing();
+
+    const winners = this.#getWinners();
+
+    Console.print(`${OUTPUT_PREFIX} ${winners}`);
+  }
+
+  async #setRacers() {
+    const firstUserInput = await Console.readLineAsync(
+      FIRST_USER_INPUT_PLACEHOLDER,
+    );
```

Comment:
```suggestion
    const userInput = await Console.readLineAsync(
      FIRST_USER_INPUT_PLACEHOLDER,
    );
```
`firstUserInput`라는 변수명이 전체적인 코드 맥락을 봤을 때는 어떤 의도로 작성해주신지 알 수 있을 것 같은데 `setRacers` 메서드만 봤을 때는 `userInput`을 한번만 받는 것 같은데 왜 `firstUserInput`이라는 변수명인지 의도를 파악하기 어려울 수 있을 것 같다고 생각이 들어서 `userInput`으로 간단하게 작성 하는건 어떻게 생각하시나요??